### PR TITLE
[ES, PT, JA, KO, RU, ZH] Translations - documentation

### PR DESCRIPTION
**Pull request by ilyaspiridonov**
Translated by Alconost

### CODE REVIEW

**Comment by ilyaspiridonov** [comment]
@devnook Please take a look

File: site/es/docs/privacy-sandbox/overview/index.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+layout: layouts/doc-post.njk
+title: What is the Privacy Sandbox?
+subhead: The Privacy Sandbox is a series of proposals to satisfy cross-site use cases without third-party cookies or other tracking mechanisms.
```
Comment (devnook) [inline site/es/docs/privacy-sandbox/overview/index.md:4]: This text should be translated

File: site/es/docs/privacy-sandbox/overview/index.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+description: "What's in it, how to get involved, and what it's for."
```
Comment (devnook) [inline site/es/docs/privacy-sandbox/overview/index.md:5]: This text should be translated

File: site/ja/docs/privacy-sandbox/attribution-reporting-migration/index.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
+---
+layout: 'layouts/doc-post.njk'
+title: コンバージョン測定APIからアトリビューションレポートAPIへの移行ガイド（Chrome 92）
+subhead: |2-
```
Comment (devnook) [inline site/ja/docs/privacy-sandbox/attribution-reporting-migration/index.md:4]: Unnecessary line + 2-

File: site/ja/docs/privacy-sandbox/overview/index.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
+---
+layout: layouts/doc-post.njk
+title: What is the Privacy Sandbox?
+subhead: The Privacy Sandbox is a series of proposals to satisfy cross-site use cases without third-party cookies or other tracking mechanisms.
```
Comment (devnook) [inline site/ja/docs/privacy-sandbox/overview/index.md:4]: Missing translation. Also on all other languages in this url.

**Comment by devnook** [comment]
There are 2 translation strings missing from each language. LGTM to eveything else.

**Comment by ilyaspiridonov** [comment]
> There are 2 translation strings missing from each language. LGTM to eveything else.

Thank you. It appears this was caused by a tab in frontmatter in [this English file]:(https://github.com/GoogleChrome/developer.chrome.com/blob/main/site/en/docs/privacy-sandbox/overview/index.md)
I've added a commit in this PR to fix that. 
The translations for these strings are also added now (had to request those separately, hence the delay).

**Comment by devnook** [comment]
lgtm

**Pull request merged**